test/syntax/run_syntax_tests.py

```
@@ -34,19 +36,25 @@ def run_syntax_tests():
-        with open(exp_path) as f:
-            expected = f.read()
+        with open(exp_path, encoding="utf-8", newline="") as f:
+            expected = f.read().replace("\r\n", "\n")
+
+        exp_rc = 0
+        rc_path = os.path.join(SCRIPT_DIR, base + ".exitcode")
+        if os.path.isfile(rc_path):
+            with open(rc_path) as f:
+                exp_rc = int(f.read().strip())
 
         result = subprocess.run([BIN, lo3_path], capture_output=True, timeout=10)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)